uzbl-tabbed.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Uzbl Tabbed
+Comment=Uzbl - web interface tools which adhere to the unix philosophy.
+Terminal=false
+Type=Application
+Icon=uzbl.png
+Exec=uzbl-tabbed %u
```

Comment:
Ah, yes. you're right. looked at the firefox one but it looked the same with a simple %u